### PR TITLE
Docker compose file 

### PR DESCRIPTION
**Pull request by nataliya-nikolova**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    environment:
+      MYSQL_ROOT_PASSWORD: my_password
+    container_name: db
+    image: mysql
+    volumes:
+      - mysql_volume:/var/lib/mysql
+    ports:
+      - 3306:3306
+    networks:
+      - network
```
Comment (mkanevv) [inline docker-compose.yml:24]: Is this working, don't we need username and password

Comment (nataliya-nikolova) [reply docker-compose.yml]: I have changed it.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    ports:
+      - '5173:5173'
+    networks:
+      - network
+
+  backend_python:
+    build: ./AI_CV_Python
+    container_name: backend_python
+    ports:
+      - '5001:5001'
```
Comment (mkanevv) [inline docker-compose.yml:41]: shouldn't it be 5000, check with Pavel

**Review by mkanevv — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (nataliya-nikolova) [reply docker-compose.yml:41]: Currently my 5000 port is used by Air Play Receiver and I don't permission to turn it down.

**Pull request merged**